core/src/packet_deserializer.rs

```diff
@@ -0,0 +1,122 @@
+//! Deserializes packets from sigverify stage. Owned by banking stage.
+
+use {
+    crate::{
+        immutable_deserialized_packet::ImmutableDeserializedPacket,
+        sigverify::SigverifyTracerPacketStats,
+    },
+    crossbeam_channel::{Receiver as CrossbeamReceiver, RecvTimeoutError},
+    solana_perf::packet::PacketBatch,
+    std::time::{Duration, Instant},
+};
+
+pub type BankingPacketBatch = (Vec<PacketBatch>, Option<SigverifyTracerPacketStats>);
+pub type BankingPacketReceiver = CrossbeamReceiver<BankingPacketBatch>;
+pub type ReceivePacketResults = (
+    Vec<ImmutableDeserializedPacket>,
+    Option<SigverifyTracerPacketStats>,
+    u64, // passed sigverify
+    u64, // failed sigverify
+);
```

Comment:
Now that we're refactoring this, it would be easier to maintain with these 2+ item tuples as structs with named fields

Reply:
Did it, nice suggestion.